app/src/main/AndroidManifest.xml

```diff
@@ -5,23 +5,23 @@
     <uses-permission android:name="android.permission.INTERNET" />
 
     <application
-        android:name=".common.ChallengesApplication"
+        android:name=".app.YoustApplication"
         android:allowBackup="false"
         android:icon="@mipmap/ic_launcher"
         android:label="@string/app_name"
         android:roundIcon="@mipmap/ic_launcher_round"
         android:supportsRtl="true"
         android:theme="@style/SharedAppTheme">
         <activity
-            android:name=".presentation.splash.SplashActivity"
+            android:name="mx.yellowme.youst.splash.SplashActivity"
```

Comment:
I think it's possible to remove `mx.yellowme.youst` from the name. 🤔

Reply:
Nice catch!